keystore.gemspec

```diff
@@ -4,7 +4,7 @@ spec = Gem::Specification.new do |s|
   s.name          = 'keystore'
   s.executables  << 'keystore.rb'
   s.license       = 'MIT'
-  s.version       = '0.1.3'
+  s.version       = '0.1.4'
```

Comment:
I had to up these version numbers tonight so this isn't gonna jive anymore. This is probably the source of the conflict.